index.html

```diff
@@ -1018,6 +1030,13 @@ <h2>
             </li>
           </ol>
         </dd>
+        <dt>
+          <dfn>type</dfn> attribute
+        </dt>
+        <dd>
+          An enumeration, {{GamepadButtonType}}, that indicates which button types the controller
+          is being held in.
```

Comment:
Is this phrasing in line with other specs? There's an [open spec issue](https://github.com/whatwg/html/issues/9832) regarding consistency in enum phrasing which makes me think this would be better phrased as:
> An enumerated {{GamepadButtonType}} attribute that classifies the current button's type in relation to an extended mapping

Reply:
updated.